src/assets/interactables/duck/gltf/DuckyMesh.json

```diff
@@ -0,0 +1,60 @@
+{
+  "scenes": {
+    "Root Scene": {
+      "shape": [
+        {
+          "shape": "box",
+          "halfExtents": {
+            "x": 0.06,
+            "y": 0.04,
+            "z": 0.08
+          },
+          "offset": {
+            "x": 0,
+            "y": 0.052671334114334445,
+            "z": 0.01001389278835843
+          },
+          "orientation": {
+            "x": 0,
+            "y": 0,
+            "z": 0,
+            "w": 1
+          }
+        },
+        {
+          "shape": "sphere",
+          "radius": 0.05,
+          "offset": {
+            "x": 0,
+            "y": 0.1287570519589527,
+            "z": 0.033095376412929145
+          },
+          "orientation": {
+            "x": 0,
+            "y": 0,
+            "z": 0,
+            "w": 1
+          }
+        },
+        {
+          "shape": "cylinder",
+          "radiusTop": 0.02,
```

Comment:
Weird that Cannon cylinders have both a top and bottom radius.. but I guess that lets you make some interesting shapes.

Reply:
Only way to get a cone